### PR TITLE
cli package code coverage

### PR DESCRIPTION
**Pull request by codyli520**
74.4% test coverage increase for cli/selfsign 
89.9% test coverage increase for cli/gencert

### CODE REVIEW

File: cli/selfsign/selfsign_test.go [hunk 1]
```diff
@@ -1 +1,31 @@
...
+	if err == nil {
+		t.Fatal("No CSR, should report error")
+	}
+
+	err = selfSignMain([]string{}, cli.Config{Hostname: ""})
+	if err == nil {
+		t.Fatal("No server, should report error")
+	}
+
+	err = selfSignMain([]string{"cloudflare.com", "../..testdata/garbage.key"}, cli.Config{Hostname: ""})
```
Comment (j-delaney) [inline cli/selfsign/selfsign_test.go]: Should `../..testdata/garbage.key` be `../../testdata/garbage.key`? (notice the added slash)

Comment (codyli520) [inline cli/selfsign/selfsign_test.go]: Thanks for the catch!

**Comment by codyli520** [comment]
Ready to be merged

**Comment by lziest** [comment]
lgtm

**Pull request merged**